micropub.php

```diff
@@ -432,18 +452,21 @@ private static function postprocess($post_id) {
       $geo = explode(':', $geo);
       $geo = explode(';', $geo[0]);
       $coords = explode(',', $geo[0]);
-      update_post_meta($post_id, 'geo_latitude', trim($coords[0]));
-      update_post_meta($post_id, 'geo_longitude', trim($coords[1]));
+      $args['meta_input']['geo_latitude'] = trim($coords[0]);
```

Comment:
you might consider adding a few unit tests in this PR, e.g. checking that post meta is stored. :P